apps/web/src/views/PositionManagers/utils/getFixedDecimals.ts

```diff
@@ -0,0 +1,29 @@
+// Position manager ROR's values can be up to very smal precision
+// this function is used to calculate nessecary decimals to show exact precision
+// for vaults with small ROR that would otherwise get rounded
+export const getFixedDecimals = (value?: number): number | undefined => {
```

Comment:
Can we use https://github.com/pancakeswap/pancake-frontend/blob/develop/packages/utils/formatNumber.ts#L61

Reply:
does the current version of this only show max precision on the tooltip? or can u specify for base display too. because right now the ROR display is already in a tooltip.